src/js/controller/captions.js

```diff
@@ -138,12 +141,13 @@ define([
             _tracksById = {};
             _metaCuesByTextTime = {};
             _unknownCount = 0;
+        }
 
-            // meta event listener may have been turned off in subtitlesTracks event
+        function _itemReadyHandler(item) {
             _model.mediaController.off('meta', _metaHandler);
             _model.mediaController.off('subtitlesTracks', _subtitlesTracksHandler);
 
-            if (!utils.isIOS()) {
+            if (_model.get('provider').name === 'flash') {
```

Comment:
We really want all providers to implement a method that tells us if they support captions tracks, and if not continue here. Since currently the only provider with this kind of support is html5, this line should be changed to:
```js
if (_model.get('provider').name !== 'html5') {
```